platform/ios/MGLMapView.mm

```diff
@@ -148,12 +148,12 @@ - (instancetype)initWithCoder:(NSCoder *)decoder
 
 - (NSString *)accessToken
 {
-    return @(_mbglMap->getAccessToken().c_str()).mgl_stringOrNilIfEmpty;
+    return @(_mbglFileSource->getAccessToken().c_str()).mgl_stringOrNilIfEmpty;
 }
 
 - (void)setAccessToken:(NSString *)accessToken
 {
-    _mbglMap->setAccessToken(accessToken ? (std::string)[accessToken UTF8String] : "");
+    _mbglFileSource->setAccessToken(accessToken ? (std::string)[accessToken UTF8String] : "");
```

Comment:
With #1553, the call to `setAccessToken()` has been moved to [`-observeValueForKeyPath:ofObject:change:context:`](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/MGLMapView.mm#L1285), which is automatically called on every instance of `MGLMapView` any time `MGLAccountManager.accessToken` changes.